README.md

```diff
@@ -596,6 +696,101 @@ This setup allows you to:
 - Provide custom documentation URLs
 - Maintain control over the OAuth flow while delegating to an external provider
 
+### Backwards Compatibility
+
+The SDK provides support for backwards compatibility between different protocol versions:
```

Comment:
This section could probably use more explanation and links, pointing out why you might care to do this.